docs/releases/5.1.txt

```diff
@@ -158,7 +158,9 @@ Minor features
 :mod:`django.contrib.syndication`
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* ...
+* All :class:`~django.utils.feedgenerator.SyndicationFeed` classes now support
+  a ``stylesheet`` attribute. If specified, an ``<? xml-stylesheet ?>``
```

Comment:
```suggestion
  a ``stylesheets`` attribute. If specified, an ``<? xml-stylesheet ?>``
```

Reply:
Doh, I forgot to update the release notes 🤦🏻 
Thanks!